src/ckb-daemon/usb_mac.c

```diff
@@ -271,6 +271,67 @@ static void pipecomplete(void* refcon, IOReturn result, void* arg0){
     (*handle)->ReadPipeAsync(handle, ctx->pipe, buffer, ctx->maxsize, pipecomplete, ctx);
 }
 
+CFMachPortRef event_tap;
```

Comment:
It must be a static variable, because no other .c file is changed yet.
Pease add it with comments in the usb.h (usb.h ist the header file for both usb_{llinux,mac}.c